internal/luhn/lunh.go

```diff
@@ -0,0 +1,24 @@
+package luhn
+
+func Valid(number int64) bool {
+	return (number%10+checksum(number/10))%10 == 0
+}
+
+func checksum(number int64) int64 {
+	var luhn int64
+
+	for i := 0; number > 0; i++ {
+		cur := number % 10
+
+		if i%2 == 0 { // even
+			cur = cur * 2
```

Comment:
```suggestion
			cur *= 2
```
можно проще чем в питоне